resources/assets/less/variables.less

```diff
@@ -18,8 +18,9 @@
 
 @import "app";
 
+@screen-mobile-max: @screen-sm-min - 1px;
 @desktop: ~"(min-width: @{screen-sm-min})";
-@mobile: ~"(max-width: @{screen-sm-min})";
+@mobile: ~"(max-width: @{screen-mobile-max})";
```

Comment:
`@media (max-width: 840px - 1px) {`
wat

need `(@screen-sm-min - 1px);` to get 839px